data/architectures.json

```diff
@@ -22,6 +22,15 @@
             "ncnn"
         ]
     },
+    "cugan": {
```

Comment:
Did you add this by hand? Because it doesn't have tags... If you didn't, then I have some bug hunting to do...